web/src/components/filtersform.js

```diff
@@ -397,6 +425,23 @@ class FiltersForm extends React.Component {
                                 placeholder="Files regexp"
                                 onChange={v => this.handleChange('files', v)}
                               />
+                              </Form.Group>
+                            <Form.Group controlId="formProjectDefinitionInput">
```

Comment:
Perhaps it would be better to wait for https://github.com/change-metrics/monocle/pull/341 and rebase on the new FilterForm component.

Reply:
Note that this PR is now rebased on the new patternfly component.